cli/medperf/entities/cube.py

```diff
@@ -220,13 +221,16 @@ def download_image(self):
             # Retrieve image from image registry
             logging.debug(f"Retrieving {self.id} image")
             cmd = f"mlcube configure --mlcube={self.cube_path}"
-            with pexpect.spawn(cmd, timeout=config.mlcube_configure_timeout) as proc:
-                proc_out = proc.read()
+            cmd_env = {**os.environ, 'DOCKER_CLI_HINTS': 'false'}
```

Comment:
this env hides `What's next?` docker CLI hits from its output, leaving only essential logs about pulling docker layers